README.md

```diff
@@ -30,28 +29,17 @@
 Introduction
 ------------
 
-**[LevelDB](https://github.com/google/leveldb)** is a simple key-value store built by Google, inspired by BigTable. It's used in Google Chrome and many other products. LevelDB supports arbitrary byte arrays as both keys and values, singular *get*, *put* and *delete* operations, *batched put and delete*, bi-directional iterators and simple compression using the very fast [Snappy](http://google.github.io/snappy/) algorithm.
+**The core of the [Level ecosystem](https://github.com/level/awesome), `levelup` is a wrapper for `abstract-leveldown` stores. These stores follow the characteristics of [LevelDB](https://github.com/google/leveldb).**
```

Comment:
Struggling with this. Suggestions?

Reply:
What about

`levelup` is a wrapper for `abstract-leveldown` compliant stores, which follow the characteristics of [LevelDB](https://github.com/google/leveldb).

We already have the awesome link from the level badge

Reply:
> We already have the awesome link from the level badge

That's true. Prefer not to open with lowercase though. Maybe just:

## Introduction

**A wrapper for `abstract-leveldown` stores. These stores follow the characteristics of [LevelDB](https://github.com/google/leveldb).**

Reply:
It's almost the same information as the short description before the badges. Does it really add anything? Maybe just tweak the first short desc?

Reply:
It doesn't really matter that much to me, just saying :)

Reply:
Added it because of the long distance to the short desc, and to have a quick segway into LevelDB. I'm probably splitting hairs ;)